### ISSUE TITLE
Guacamole RDP doesn't work with encrypted CONTEXT/PASSWORD

### ISSUE THREAD

**Issue by feldsam** [OpenNebula/one#5871]
**Description**
CLIENT_ERROR when CONTEXT/PASSWORD is encrypted

**Details**
 - Affected Component: [Sunstone]
 - Hypervisor: [KVM]
 - Version: [6.*]

## Progress Status
- [x] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by FrederickBor** [comment]
Hi @feldsam,

We cannot reproduce this issue on the development version or v6.4.
Could you please provide a step-by-step guide to reproduce the problem? 

Regards,
Frederick

**Comment by feldsam** [comment]
Hello, I tested it on 6.2.2. I set `VM_ENCRYPTED_ATTR = "CONTEXT/PASSWORD"`  in oned.conf. I spinup VM with user variables `USERNAME` and `PASSWORD`. When I debug what is returned by `retrieve_resource` method, I see `CONTEXT/PASSWORD` encrypted. When I set third param to true, it returns VM data with decrypted `CONTEXT/PASSWORD`. When I investigate code, there is `retrieve_resource` method which calls `info` method on the resource

https://github.com/OpenNebula/one/blob/63eb2447db602a964029441a6b77ad7735a953d2/src/sunstone/models/SunstoneServer.rb#L627

in this case, it is `virtual_machine` resource. When I look on `info` method in this resource, I see that first and one argument  sets if data should be decrypted or not

https://github.com/OpenNebula/one/blob/bf6ef68bd15a825e5ccb1ce3602f868257d6a22a/src/oca/ruby/opennebula/virtual_machine.rb#L296

and it is actually working with my simple change.
